napari/layers/_image_layer/model.py

```diff
@@ -326,6 +332,34 @@ def interpolation_functions(self):
     def _clim_range_default(self):
         return [float(self.image.min()), float(self.image.max())]
 
+    def _update_thumbnail(self):
+        """Update thumbnail with current image data and colormap.
+        """
```

Comment:
I've renamed that first one `zoomed` instead of `thumbnail` as it was a little too soon to use that word, it corresponds to the downsampled image. `mapped` corresponds to the result after mapping through colormaps. i can improve if still confusing

Reply:
Isn't a more accurate name "downsampled?"

Reply:
yup, better choice - will fix